### ISSUE TITLE
Controller / input bug introduced in Yuzu EA 2733 (+)

### ISSUE THREAD

**Issue by famouslut** [yuzu-emu/yuzu#8422]
I've noticed a "sticky" controller bug that was introduced in Yuzu EA 2733 and all subsequent versions, seems to affect vulkan mode _only_. I use an Xbox1 S (bluetooth) ctrlr (switch pro ctrlr mode) and the bug possibly won't be noticed by many users; difficult to reproduce in (eg) Metroid Prime. But does happen universally.

To (best / quickly) reproduce, load MK8 and try to complete a lap (and use powerslide - ie triggers) The directional (l-stick) input gets "locked" sometimes. EA 2732 works as expected, so any changes introduced after that are where the issue originates. Log probably won't help, it happens (merely) very often, not at a specific point.

Seems fine w/ joycon. Although using 2x joycons (in "pro" ctrl mode) seems to give (very intermittent) issues, as above?

**Comment by german77** [comment]
Disable vibrations and let me know if that fixes anything

**Comment by famouslut** [comment]
Yeah, that works as a fix, albeit more a workaround! Thx! Annoying that you can't get the feedback (vibration) now :(

**Comment by german77** [comment]
I'm trying to see where it went wrong

**Comment by famouslut** [comment]
Hopefully it's an easy fix! I've noticed that it doesn't necessarily depend on stick / trigger input; sometimes just pressing / holding (eg) B / A button will cause a "wild" stick "amplification". Almost as if you jammed the l-stick to the max left / right? (still happens in EA 2757)

**Comment by german77** [comment]
The fix just dropped today

**Comment by famouslut** [comment]
Great job, hope it wasn't too hard to trace! Thanks again.

**Comment by famouslut** [comment]
Issue appears again in EA 2818 :(

**Comment by german77** [comment]
That's weird I'm pretty sure that has the fix. Are you able to confirm by testing the older EA. Also what was the number of the old EA?

**Comment by famouslut** [comment]
It was fine in EA 2816, yesterday, regression in 2818. I tried (quickly) to test 2816, but it doesn't seem like the fix works in that, either, now.

**Comment by benderscruffy** [comment]
Has this been fixed ???
Please update

**Comment by unclesammy** [comment]
> Has this been fixed ??? Please update

I'm buying a new pair of joycon just to fix this problem.But obviously that's not the problem.I still received frame droped using the joycon.Do I need to buy a Bluetooth 4.0?
